install_dependencies.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -ex
+# export BUILD_ARCHITECTURE=arm64
+# Get ubuntu distribution code name. All STOL APT debian packages are pushed to S3 bucket based on distribution codename.
+# Install rapidjson commit instead of release since there has been no new release since August 2016
+# but repo is still under active development
+
+# /opt/carma/scripts/install_dependencies_script.sh 
```

Comment:
Is this used?

Reply:
Seems like this file is optional https://github.com/usdot-fhwa-stol/actions/blob/bc0492c1eded273091827845438433a523e0387c/.github/workflows/cpp-module-build.yml#L68  and should be fine to remove